benchmarking/benchmarks/kairo/kairo_avoidable.js

```diff
@@ -53,7 +53,38 @@ export async function kairo_avoidable() {
 	destroy();
 
 	return {
-		benchmark: 'kairo_avoidable',
+		benchmark: 'kairo_avoidable_unowned',
+		time: timing.time.toFixed(2),
+		gc_time: timing.gc_time.toFixed(2)
+	};
+}
+
+export async function kairo_avoidable_owned() {
+	// Do 10 loops to warm up JIT
+	for (let i = 0; i < 10; i++) {
+		const { run, destroy } = setup();
```

Comment:
should this also use `effect_root` already to be more in line with which code paths are later used?

Reply:
Good point.